students/MzKhan/lesson03/mailroom.py

```diff
@@ -0,0 +1,153 @@
+'''
+    Name: Muhammad Khan
+    Date: 02/28/2019
+    Assignment03
+
+'''
+
+#Goal:
+# Automate the mailroom job of an individual working at a local
+# charity organization. The script will prompt the user for the varioius options
+# and perform the desired tasks.
+
+# Global donors List.
+# This list is accessible everywhere in the script.
+
+donors_list = [ ['Adam', 'Johnson', 6000.20, 2200, 1050,100],
+              ['Matt', 'Marvin', 550,1000,250],
+              ['Ashley', 'Wiggins', 55.66,270,1000],
+              ['Kristina', 'Laughrey', 500, 500,200],
+              ['Kimberley', 'Allen', 12000.99, 5000],
+              ['Doug', 'Boolinger', 260.57, 930.90],
+              ['Sherry', 'Henning', 6000, 2460.20, 900] ]
+
+def prompt_user():
+    # This method prompts the user for the valid input.
+    # input: string
+    # return: int
+    print('Please choose one of the following options: \n')
+    print("1: Send a Thank You")
+    print("2: Create a report")
+    print("3: Quit")
+    return int(input("Option:  "))
+
+def thank_you(donors_data = donors_list):
```

Comment:
in general using mutable data structure for default value is a bad practice:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html